Test/pruning/UsesClauses.bpl.pruned.before-doesNotHaveAxioms.expect

```diff
@@ -0,0 +1,53 @@
+const unique four: int;
+
+const unique ProducerConst: bool uses {
+  axiom four == 4;
+}
+
+axiom four == 4;
+
+axiom four == 4;
+
+axiom four == 4;
```

Comment:
How come these axioms are printed so many times, included inside a `uses` clause (on L4)?

Reply:
That's a bug